src/include/sof/audio/component.h

```diff
@@ -338,7 +338,7 @@ struct comp_ops {
 	int (*dai_config)(struct dai_data *dd, struct comp_dev *dev,
 			  struct ipc_config_dai *dai_config, const void *dai_spec_config);
 
-#if CONFIG_IPC_MAJOR_3
+#if CONFIG_IPC_MAJOR_3 || CONFIG_LIBRARY
```

Comment:
@ranj063 could you submit this one-line fix in a separate PR so it can be fast-tracked?